client/layout/index.jsx

```diff
@@ -100,6 +101,46 @@ class Layout extends Component {
 		// intentionally don't remove these in unmount
 	}
 
+	renderMasterbar() {
+		const MasterbarComponent = config.isEnabled( 'jetpack-cloud' )
+			? JetpackCloudMasterbar
+			: MasterbarLoggedIn;
+
+		return (
+			<MasterbarComponent
+				section={ this.props.sectionGroup }
+				isCheckout={ this.props.sectionName === 'checkout' }
+			/>
+		);
+	}
+
+	shouldLoadInlineHelp() {
```

Comment:
This should probably include the `sectionName` of `devdocs` as well (see #34360)